### PR TITLE
Add modal support

### PR DESCRIPTION
**Pull request by bqqbarbhg**

### CODE REVIEW

**Comment by bqqbarbhg** [comment]
Someone needs to generate `package-lock.json` and commit it since I have Windows!

**Comment by HenrikHelenius** [comment]
🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄

**Review by HenrikHelenius — APPROVED** [review]
🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄

File: client/src/shadowDomHelper.js [hunk 1]
```diff
@@ -6,15 +6,25 @@ const shadowRoot = () => document
 const shadowDocument = () => document
   .querySelector('[data-feedback-shadow-root]').shadowRoot
 
+const shadowModalRoot = () => document
+  .querySelector('[data-feedback-shadow-root]').shadowRoot
```
Comment (HenrikHelenius) [inline client/src/shadowDomHelper.js]: use shadowDocument()

**Review by HenrikHelenius — DISMISSED** [review]

**Review by kalomak — APPROVED** [review]
🤤🤤🤤

**Pull request merged**